pages/home_page.py

```diff
@@ -36,20 +33,21 @@ def wait_for_page_to_load(self):
 
     def click_last_product_top_crashers_link(self):
         return self.ReleaseChannels(
-            self.testsetup, self.selenium.find_element(*self._last_release_channel_locator)
+            self.timeout, self.selenium, self.selenium.find_element(*self._last_release_channel_locator)
```

Comment:
I think `self.timeout` here should be `self.base_url` and `ReleaseChannels` should extend `PageRegion` rather than `object`. That will change much of the code below, eliminating the need for the `__init__` in `ReleaseChannels`, for example.